CHANGELOG.md

```diff
@@ -23,9 +23,16 @@
 * Inline grid editing supports passing of JSX editor components.
 * `GridExportService` catches any exceptions thrown during export preparation and warns the user
   that something went wrong.
+* GridModel with 'disabled' selection no longer shows "ghost" selection when using keyboard.
 * Tree grids now style "parent" rows consistently with highlights/borders if requested, even for
   mixed-depth trees where some rows have children at a given level and others do not.
 
+### 💥 Breaking Changes
+* The raw `AgGrid` component no longer enhances the native keyboard handling provided by ag-Grid.
+All Hoist key handling customizations are now limited to `Grid`.  If you wish to provide custom
+handling in a raw `AgGrid` component, see the example here:
+     https://www.ag-grid.com/javascript-grid/row-selection/#example-selection-with-keyboard-arrow-keys
```

Comment:
If we want to keep this in a 41.2.0 release, I'd vote for making this a "⚠" note inline with the bugfix bullet point.

That way only major version releases have a "Breaking Changes" header, with these occasional "edge case breaking changes" being handled differently, but consistently as call-outs in the other sections.